mlflow/openai/_openai_autolog.py

```diff
@@ -200,9 +200,9 @@ def _stream_output_logging_hook(stream: Iterator) -> Iterator:
                 # so that the model can be logged.
                 with _set_api_key_env_var(self._client):
                     mlflow.openai.log_model(
-                        kwargs.get("model", None),
-                        task,
-                        "model",
+                        model=kwargs.get("model", None),
```

Comment:
Can we update this to link the traces to the model as well? See the demo notebook for an example.

Reply:
got it, will check the notebook

Reply:
https://github.com/mlflow/mlflow/blob/a049494a66bee10267435566c01a4b6cb0a2bc15/mlflow/tracing/fluent.py#L192